### PR TITLE
Tuto for windows users

### PR DESCRIPTION
**Pull request by fteytaud**
## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Solving issue #592.
I am not at all a windows expert, but I try to do something as simple as possible.

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,9 @@ Learn more on parametrization in the [**documentation**](https://facebookresearc
 
 Check out our [**documentation**](https://facebookresearch.github.io/nevergrad/)! It's still a work in progress, don't hesitate to submit issues and/or PR to update it and make it clearer!
 
+## Windows installation
+
+For windows installation, please refer to [the windows documention](docs/windows.md)
```
Comment (jrapin) [inline README.md:74]: Most of the install intructions are in getting started, you should at least add a link there as well.
I would tend to remove the lines here and update line 14 to state that Windows install can be found there

Comment (fteytaud) [reply README.md]: I have reduced all images, it was quite ugly (not sure this is beautiful now, I think I am not good at this :) ).
I have corrected the links, as you suggested. I am not familiar to rst, so I just add a 

```
Installing on Windows
----------

For windows installation, please refer to `the windows documention <https://github.com/facebookresearch/nevergrad/blob/master/docs/windows.md>`
```
in the getting_started.rst file. Not sure if this is a good way to do it :)
(and I can't check the link)

**Comment by jrapin** [comment]
I've made a PR on your PR to update the links: https://github.com/fteytaud/nevergrad/pull/1
After that it should be ok ;) Thanks!
I'll probably update the document eventually though, to use rst, and maybe move the PR part to the contributing section. But that will wait for now ;)

**Comment by fteytaud** [comment]
Ok thx. Let me know if you need something.

**Comment by jrapin** [comment]
> Ok thx. Let me know if you need something.

Just merge my PR into yours (should work by just clicking "merge" in  https://github.com/fteytaud/nevergrad/pull/1)

**Pull request merged**